### ISSUE TITLE
Issue With gradle sync

### ISSUE THREAD

**Issue by poovarasanvasudevan** [flipkart-incubator/proteus#89]
Error:Module 'com.github.flipkart-incubator.proteus:design:5.0.0-rc8' depends on one or more Android Libraries but is a jar
`
compile 'com.github.flipkart-incubator.proteus:design:5.0.0-rc8'

`

![image](https://cloud.githubusercontent.com/assets/8036283/25609760/3bce575c-2f3e-11e7-8151-c1c16efff9ab.png)

**Comment by adityasharat** [comment]
This seems to be an issue in the way we are packaging proteus in jitpack. To fix this please use the following:

```
 compile('com.github.flipkart-incubator.proteus:proteus-core:5.0.0-rc8@aar') { changing = true }
    compile('com.github.flipkart-incubator.proteus:gson-adapter:5.0.0-rc8@aar') { changing = true }
    compile('com.github.flipkart-incubator.proteus:recyclerview-v7:5.0.0-rc8@aar') { changing = true }
    compile('com.github.flipkart-incubator.proteus:cardview-v7:5.0.0-rc8@aar') { changing = true }
    compile('com.github.flipkart-incubator.proteus:design:5.0.0-rc8@aar') { changing = true }
```

**Comment by adityasharat** [comment]
Closed in favour of #118
